src/AhRulesBot/AhRulesBot/BotRequestsProcessing/MessageValidator.cs

```diff
@@ -46,21 +54,49 @@ public async Task<bool> IsValid(Message message)
             if (DateTime.UtcNow.Subtract(message.Date).TotalMinutes > 4)
                 return false;
 
+            if (IsSpamming(message))
+            {
+                return false;
+            }
+
             return true;
         }
 
+        private bool IsSpamming(Message message)
```

Comment:
pls add tests (may be not in the pr, but later)

Reply:
Thx, I will